docs/en/operations/settings/settings-formats.md

```diff
@@ -1102,6 +1102,12 @@ Use Parquet FIXED_LENGTH_BYTE_ARRAY type instead of Binary/String for FixedStrin
 
 Enabled by default.
 
+### output_format_parquet_version {#output_format_parquet_version}
+
+The version of Parquet format used in output format. Supported versions: `v1_0`, `v2_4`, `v2_6` and `v2_latest`.
```

Comment:
Is `v` and underscore natural or expected?

Why don't we use `2.4`, `2.latest`?

Reply:
I just used same names as it's in arrow parquet library. But let's change to version with dots.

Reply:
```suggestion
The version of Parquet format used in output format. Supported versions: `1.0`, `2.4`, `2.6` and `2.latest`.
```